db/migrate/20150408012314_remove_beer_from_users.rb

```diff
@@ -1,5 +1,5 @@
 class RemoveBeerFromUsers < ActiveRecord::Migration
   def change
-    remove_column :users, :beer, :string
+    # remove_column :users, :beer, :string
```

Comment:
I would delete this migration if it's not being used. If it has already been run by other team members though then don't change it at all.